cmd/server/gpg/gpg.go

```diff
@@ -0,0 +1,87 @@
+// THIS IS BORROWED FROM FLUX!
```

Comment:
Can we add a reference to Flux's implementation? It's apache 2.0 licenced, so there is nothing wrong with using it.
It's just nice to have a link back to the original.